### DISCUSSION TITLE
Post crushes

### DISCUSSION THREAD

**Discussion by martienne17** [AprilSylph/XKit-Rewritten#1242] [Suggestion Box]
Just wondered if you'd bring back [post crushes](https://www.tumblr.com/tagged/post%20crushes?sort=recent)! I loved that feature of x-kit.

**Comment by AprilSylph** [comment]
Aren't crushes currently broken on Tumblr's side? Mine haven't moved in a few years.

**Reply by martienne17** [reply]
Oh, I have no idea. I don't even know how it works lol
